models/users.ts

```diff
@@ -31,6 +31,58 @@ import { Schema } from 'mongoose';
  @property {String} photo - links to photo files uploaded by user
  */
 
+export interface UserResponse {
+  _id: string;
+  active?: boolean;
```

Comment:
seems that active is mandatory option

Reply:
yes. but I added it for a future features. Now this option doesn't use.